nsv13/code/modules/overmap/ai.dm

```diff
@@ -69,4 +122,14 @@
 	if(OM.tactical)
 		var/sound = pick('nsv13/sound/effects/computer/alarm.ogg','nsv13/sound/effects/computer/alarm_3.ogg','nsv13/sound/effects/computer/alarm_4.ogg')
 		var/message = "<span class='warning'>DANGER: [src] is now targeting [OM].</span>"
-		OM.tactical.relay_sound(sound, message)
\ No newline at end of file
+		OM.tactical.relay_sound(sound, message)
+	try_launch_fighters()
+
+/obj/structure/overmap/proc/try_launch_fighters()
+	if(ai_can_launch_fighters) //Found a new enemy? Launch the CAP.
```

Comment:
move this check into the /obj/structure/overmap/syndicate/ai/carrier/ai_target( and create a child of the target proc above for the carrier which literally adds the try_launch_fighters after inheritance with the check too. 

The reasoning is that each time its processing the AI it will go into this proc for the carrier when it doesnt need to. 
(wastes processing time)
Nothing else after that calls this proc. Your safe to remove the check inside the proc.